google/oauth2/id_token.py

```diff
@@ -59,6 +59,13 @@
 import json
 import os
 
+try:
+    import jwt as jwt_lib  # type: ignore
```

Comment:
We have to try importing only if it is required. If not we will be breaking existing users by trying to import and raise the exception.